### DISCUSSION TITLE
2일차 파이썬 코드 링크 받아보고 싶습니다.

### DISCUSSION THREAD

**Discussion by woong6642** [Road-Balance/gcamp_ros2_basic#47] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/109203706/178943545-6c215172-6cf3-4024-b864-e3e78e30f64f.png)

링크가 눌려지지 않는데 링크 받아보고 싶습니다

**Comment by kimsooyoung** [comment, marked as answer]
링크 공유드립니다.

1. https://dojang.io/mod/page/view.php?id=2448
2. https://dojang.io/mod/page/view.php?id=2372
